examples/ex1.c

```diff
@@ -0,0 +1,70 @@
+
+static char help[] = "Trains binary SVM classification model using precomputed Gramian (linear kernel) matrix.\n\
+                      Provided Gramian matrix is computed and related to no-bias SVM formulation (svm_binary_mod 2).\n";
```

Comment:
Remove mention of the precomputed Gramian (it is not neccessary to set it). Remove second line (any kernel file can be used). Describe input parameters as in e.g.,  examples/exbinfile.c.

Reply:
Good point! It makes sense to write the example more generally. I agree. Accepted.